### ISSUE TITLE
Content: Clarify COE Dates for Students in Status Cards 

### ISSUE THREAD

**Issue by HRAGANBC** [bcgov/SIMS#2616]
**Description:**
Revise content wording to clarify that Institutions cannot confirm enrolment until 21 days prior to the student's start date. 

**Current Behaviour:**
 - Insert photo of Application Status Tracker card explaining that Schools must confirm enrolment
  
<img width="806" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBekM4QWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--38884b261bd00b676771806e8abdb59614a50d89/image.png" alt="image.png" />

 
**Acceptance Criteria**
- [ ] Revise wording on application status tracker card to read: " **Waiting for your confirmation of enrolment** Your school will be confirming your enrolment. Your school cannot confirm your enrolment until 21 days prior to your study start date. You can contact your Financial Aid Officer from your school if you require more information."

**Comment by michesmith** [comment]
@astridSABC will create a new ticket once Jeremy is done his content review

**Comment by michesmith** [comment]
@astridSABC will create a new ticket once Jeremy is done his content review
